frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/health-card/utils.ts

```diff
@@ -21,12 +23,18 @@ const CephHealthStatus = [
   },
 ];
 
-export const getCephHealthState = (ocsResponse): CephHealth => {
-  if (!ocsResponse) {
+export const getCephHealthState = (ocsResponse, cephCluster): CephHealth => {
+  if (!ocsResponse || !cephCluster) {
```

Comment:
```suggestion
  if (!ocsResponse || !_.get(cephCluster, 'loaded')) {
```

other than that, lgtm. Once https://github.com/openshift/console/pull/2480 is merged, please rebase and we can merge